### DISCUSSION TITLE
YetiForce 7.0 release

### DISCUSSION THREAD

**Discussion by paula-w** [YetiForceCompany/YetiForceCRM#17427] [Announcement]
Hello everyone!

### We’re thrilled to announce that the **long awaited version 7.0 is now available**. 
Below, you can find all the necessary links to download the latest stable version, test it, and find out what’s new.

- [DOWNLOAD]( https://api.yetiforce.eu/download/crm/www/7.0.1-complete)
- [DEMO](https://demo.yetiforce.com/)
- [FULL CHANGELOG](https://doc.yetiforce.com/introduction/release-notes-changelog/)
- [LICENSE](https://yetiforce.com/en/license.html)

The 7.0 version introduces over **100 key changes** to improve your system; some of the most requested improvements include:
- A new modern interface
- An advanced Reports module
- Single Sign-On functionality
- OAuth 2.0 support
- New integrations and improvements for the existing ones

**As always, huge thanks to our amazing community for all your support ❤.**

Happy testing!

**Comment by JohnMaverick** [comment]
A bit more restrictive license... gone through as follows:

Commercial Use Limitations: The license restricts the deployment of the software for commercial purposes unless carried out by an authorized partner or the entity itself for in-house use. Open-source licenses typically do not have such restrictions.

Registration Requirement: The license requires that the software must be registered using a built-in form with an active internet connection. Open-source software usually does not have such requirements and allows for anonymous use.

Modification and Redistribution Restrictions: The license specifies that modifications for commercial services or redistribution for commercial purposes require the producer's consent. Open-source licenses generally allow modifications and redistribution without the need for additional permissions.

Decompiling Prohibitions: The license prohibits the decompiling or decrypting of the software. Open-source software generally allows users to access and modify the source code freely.

Usage of Proprietary Code: The software incorporates code from vtiger and sugarcrm, which are open source but under specific licenses that must be complied with. The usage of these codes does not inherently make the software open source unless it adheres to the open-source definitions typically associated with those licenses.

Based on these observations, while the software may incorporate open-source components and be free of charge for certain uses, it imposes restrictions that are not consistent with the principles of open-source software as defined by the Open Source Initiative (OSI). Thus, it should be considered as proprietary software with some elements of open-source code.

**Reply by JohnMaverick** [reply]
It doesnt means it is not cool, as only reporting module makes huge change. Obviously testing will take lots of time.

**Comment by FranziskaSom** [comment]
have the passwords disappeared?
or are they simply not activated?
it would also be cool to be able to view the settings with 0 rights ...

what happened to assets, weren't they separate?

**Reply by mariuszkrzaczkowski** [reply]
some modules are inactive
https://demo.yetiforce.com/index.php?module=ModuleManager&parent=Settings&view=List

**Reply by FranziskaSom** [reply]
hi :) and thanks for answer ...
i/we cant see the link 

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/75488133/8822afda-ebdf-4dd9-a345-a2cfab849140)

**Reply by mariuszkrzaczkowski** [reply]
Login admin pass admin 🤪

**Reply by FranziskaSom** [reply]
🤣🤣🤣

**Reply by mariuszkrzaczkowski** [reply]
it works ?

**Reply by FranziskaSom** [reply]
Yes :) thanks ...

**Comment by djimatr** [comment]
Window scrolling is not possible during installation - its been disabled on Chrome at least c heck your javascript - lots of errors on the install

**Comment by MonsterITServices** [comment]
pdo_sqlsrv and sqlsrv for php8.1 does not get installed anymore. (major problems with it) so you might want to allow php7.4 to run as normal unless you can do a wiki on how to activate it.

**Reply by mariuszkrzaczkowski** [reply]
this is optional
`pdo_sqlsrvpdo_sqlsrv - Library required if integration with Wapro ERP or Comarch is active`

**Reply by MonsterITServices** [reply]
Okay, however the cron errors if not installed.

**Comment by FranziskaSom** [comment]
i don't see any ideas in the demo. are they still there?

**Reply by mariuszkrzaczkowski** [reply]
https://demo.yetiforce.com/index.php?module=ModuleManager&parent=Settings&view=List
![Screenshot_20240423_214400_Chrome](https://github.com/YetiForceCompany/YetiForceCRM/assets/10198654/f99aac03-59e6-4e99-91be-b74d4d513d42)

**Comment by gmisiolek-sbm** [comment]
I would like to ask if is the repository for version 7.x publically available? I see no new changes are added to https://github.com/YetiForceCompany/YetiForceCRM

I like this release, new layout looks clean. I've added AzureAD integration to our on-premise instance, it works good, but without documentation it was quite tricky ;)

If I may suggest something about this Azure AD integration:
Your implementation force that only users with real mail inbox can use Azure AD as authentication source. I've tested on AzureAD user that has no license assigned, hence this user has no access to inbox, but his login in Azure AD is e-mail address. And in that case, there is no `email` in payload from provider, nor `mail` in response from microsoft graph. 
I think it would be great if you will add checking `$userInfo['userPrincipalName']` from microsoft graph, as it is username as e-mail address.

**Reply by gmisiolek-sbm** [reply]
@paula-w any response on it? Also, if I have found a bug in version 7, where can I post it or add pull request with change?

**Comment by JohnMaverick** [comment]
Would be of and advantage if some popular VM with nicely configured OS and installed yeti will be available in this repo.

**Comment by shawkatmarrouf** [comment]
It would be great to add new records import mechanism to import data from excel directly with validation process
